build.sbt

```diff
@@ -37,11 +39,17 @@ bintrayOrganization := Some("localytics")
 bintrayPackageLabels := Seq("localytics", "sbt", "aws", "s3", "test", "testing")
 
 // Error on conflicting dependencies
-// http://www.scala-sbt.org/0.13/docs/Library-Management.html#Conflict+Management
+// http://www.scala-sbt.org/1.x/docs/Library-Management.html#Conflict+Management
 conflictManager := ConflictManager.strict
 
 // Error on circular dependencies
-// http://www.scala-sbt.org/0.13/docs/sbt-0.13-Tech-Previews.html#Circular+dependency
+// http://www.scala-sbt.org/1.x/docs/sbt-0.13-Tech-Previews.html#Circular+dependency
 updateOptions := updateOptions.value.withCircularDependencyLevel(CircularDependencyLevel.Error)
 
-libraryDependencies += "org.scalatest" %% "scalatest" % "2.2.6" % "test"
+libraryDependencies +=
+  ("org.scalatest" %% "scalatest" % "3.0.4" % "test"
+    // Scalatest 3.0.4 is slightly behind sbt and Scala dependencies for these two modules, so they must be excluded
+    // while Strict dependency checking is enabled.
+    exclude("org.scala-lang.modules", s"scala-xml_${scalaBinaryVersion.value}")
```

Comment:
This is a dumb question but can you use the `%%` syntax here? Like

`exclude("org.scala-lang.modules" %% "scala-xml")`

or is that just a thing that would make sense in a world where sbt makes sense.

Reply:
I gave this a try but sadly it does not compile.

Reply:
Ah, my bad, it's `excludeAll("org.scala-lang.modules" %% "scala-xml")`

Reply:
(the above compiles--I'm assuming if it compiles it works: that's how it goes with Scala, right? 😁 )